libnopegl/src/nopegl.h.in

```diff
@@ -102,14 +102,28 @@ struct ngl_node;
 /**
  * Scene graph and its rendering parameters
  */
-struct ngl_scene {
-    void *internal;         /* reserved internal pointer */
+struct ngl_scene;
+
+struct ngl_scene_params {
     struct ngl_node *root;  /* root node of the scene graph */
     double duration;        /* for how long the scene is to be rendered, in seconds */
     int32_t framerate[2];    /* fraction representing the expected rendering number of frames per seconds */
     int32_t aspect_ratio[2]; /* fraction representing the expected rendering aspect ratio */
 };
 
+/**
+ * Return a ngl_scene_params struct filled with the defaults.
+ *
+ * Default parameters:
+ * - duration: 30 seconds
+ * - framerate: 60 FPS
+ * - aspect ratio: 1:1
+ *
+ * A NULL root pointer is supported if it is not yet known when calling this
+ * function.
```

Comment:
I think you can just do something like:
```@params a pointer to the root node of a graph (can be NULL)```
And also while at it add a `@return` entry so it stays consistent with the other comments.

Reply:
done:
```diff
diff --git a/libnopegl/src/nopegl.h.in b/libnopegl/src/nopegl.h.in
index 4285a6990..14f39d772 100644
--- a/libnopegl/src/nopegl.h.in
+++ b/libnopegl/src/nopegl.h.in
@@ -112,15 +112,14 @@ struct ngl_scene_params {
 };

 /**
- * Return a ngl_scene_params struct filled with the defaults.
- *
- * Default parameters:
+ * Get the default parameters for the scene initialization:
  * - duration: 30 seconds
  * - framerate: 60 FPS
  * - aspect ratio: 1:1
  *
- * A NULL root pointer is supported if it is not yet known when calling this
- * function.
+ * @param root pointer to the root node of a graph (can be NULL)
+ *
+ * @return a ngl_scene_params struct filled with the defaults
  */
 NGL_API struct ngl_scene_params ngl_scene_default_params(struct ngl_node *root);
```